core/web/resolver/feeds_manager.go

```diff
@@ -180,16 +173,18 @@ func (r *FeedsManagersPayloadResolver) Results() []*FeedsManagerResolver {
 // CreateFeedsManagerPayloadResolver
 type CreateFeedsManagerPayloadResolver struct {
 	mgr *feeds.FeedsManager
-	err error
 	// inputErrors maps an input path to a string
 	inputErrs map[string]string
+	NotFoundErrorUnionType
 }
 
 func NewCreateFeedsManagerPayload(mgr *feeds.FeedsManager, err error, inputErrs map[string]string) *CreateFeedsManagerPayloadResolver {
+	e := NotFoundErrorUnionType{err, "feeds manager not found", nil}
+
 	return &CreateFeedsManagerPayloadResolver{
-		mgr:       mgr,
-		err:       err,
-		inputErrs: inputErrs,
+		mgr,
+		inputErrs,
+		e,
```

Comment:
Declare the field names. Using unnamed fields provides less clarity and is annoying to deal with when the struct field ordering changes.

```suggestion
		mgr: mgr,
		inputErrs: inputErrs,
		NotFoudnErrorUnionType: e,
```